docs/source/mps_environment_variables.rst

```diff
@@ -0,0 +1,40 @@
+.. _mps_environment_variables:
+
+MPS Environment Variables
+==========================
+
+**PyTorch Environment Variables**
+
+.. list-table::
+  :header-rows: 1
+
+  * - Variable
+    - Description
+  * - ``PYTORCH_DEBUG_MPS_ALLOCATOR``
+    - If set to ``1``,set allocator logging level to verbose.
+  * - ``PYTORCH_MPS_HIGH_WATERMARK_RATIO``
+    - High watermark ratio for MPS allocator. By default, it is set to 1.7.
+  * - ``PYTORCH_MPS_LOW_WATERMARK_RATIO``
+    - Low watermark ratio for MPS allocator. By default, it is set to 1.4 if the memory is unified and set to 1.0 if the memory is discrete.
+  * - ``PYTORCH_MPS_FAST_MATH``
+    - If set to ``1``, enable fast math for MPS metal kernels. See section 1.6.3 in https://developer.apple.com/metal/Metal-Shading-Language-Specification.pdf for precision implications.
```

Comment:
Hmm, this is not aligned with what you've defined in previous API, where setting it to zero would also enable fast-math, wouldn't it?

Reply:
Ah let me rectify it!